validphys2/src/validphys/n3fit_data.py

```diff
@@ -74,14 +74,17 @@ def __iter__(self):
             yield m
 
 
-def tr_masks(data, replica_trvlseed):
+def tr_masks(data, replica_trvlseed, parallel_models=False, replica=1, replicas=(1,)):
     """Generate the boolean masks used to split data into training and
     validation points. Returns a list of 1-D boolean arrays, one for each
     dataset. Each array has length equal to N_data, the datapoints which
     will be included in the training are ``True`` such that
 
         tr_data = data[tr_mask]
 
+    The single_datapoints_toss flag signals whether single-point datasets
```

Comment:
I think this was never implemented?

Reply:
Can confirm that this doesn't appear anywhere else, but I'm not sure what this is about, if the comment can just be removed or if something more needs to be implemented.